.github/workflows/build_flang_arm64.yml

```diff
@@ -36,18 +35,20 @@ jobs:
           cmake --version
           make --version
 
-      - name: Extract SHA
-        run: echo "::set-output name=sha::${GITHUB_SHA}"
-        id: extract
+      - name: Manual checkout to build in user's home dir (push)
+        if: github.ref == 'refs/heads/main'
```

Comment:
There is no `main` branch in https://github.com/flang-compiler/flang/. Is this intentional or a bug?